cashctrl_ledger/ledger.py

```diff
@@ -45,13 +45,10 @@ def dump_to_zip(self, archive_path: str):
                     default_settings[key] = self._client.account_from_id(system_settings[key])
             settings["DEFAULT_SETTINGS"] = default_settings
 
```

Comment:
Why do we need a`dump_to_zip` method in CashCtrlLedger?

I believe the parent class `LedgerEngine.dump_to_zip()` method will work for everything except settings.
I would implement a method `LedgerEngine.list_settings() -> dict` that is invoked inside `LedgerEngine.dump_to_zip()` and can be overridden in CshCtrlLedger for the specific purpose of dumping the settings.

(Btw: `LedgerEngine.list_settings()` could become a future `LedgerEngine.settings.list().`